tools/docker/windows/install_intellij.sh

```diff
@@ -0,0 +1,46 @@
+#!bin/bash
+
+ed=C
+
+# Fetch the most recent version
+VERSION=$(wget "https://www.jetbrains.com/intellij-repository/releases" -qO- | grep -P -o -m 1 "(?<=https://www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/idea/BUILD/)[^/]+(?=/)")
+
+# Prepend base URL for download
+URL="https://download.jetbrains.com/idea/ideaI$ed-$VERSION.tar.gz"
```

Comment:
I have no idea why you use that `$ed` thing.

Reply:
taken from other sources